setup.py

```diff
@@ -0,0 +1,16 @@
+from distutils.core import setup
```

Comment:
Small request: should we maybe document at `README.md` how to use this? :-)

Yeah, even if it's maybe quite obvious.

Reply:
I would add an issue for improving the README if that is ok 😉.

Reply:
https://github.com/uyuni-project/terracumber/issues/16